src/client/hooks/useThemaMenuItems.ts

```diff
@@ -19,18 +19,35 @@ export interface ThemasState {
   isLoading: boolean;
 }
 
+type withIDTitle = { id: string; title: string };
+const sortAlphaOnTitle = sortAlpha('title');
+
+export function compareThemas<T extends withIDTitle>(a: T, b: T): 0 | 1 | -1 {
+  // These will be placed on top in the order they are put here.
+  const themaIDsOnTop = [themaIdBRP, themaIdKVK] as string[];
+
+  const aHasPrecedence = themaIDsOnTop.includes(a.id);
+  const bHasPrecedence = themaIDsOnTop.includes(b.id);
+
+  if (aHasPrecedence && bHasPrecedence) {
+    const ia = themaIDsOnTop.indexOf(a.id);
+    const ib = themaIDsOnTop.indexOf(b.id);
+    return ia < ib ? -1 : 1;
+  }
```

Comment:
Volgens mij kan dit ook. Dan heb je deze en de laatste. Maarja mogelijk leesbaarder om het op te splitsen. Zou hier wel sortAlphaOnTitle gebruiken, want basically doe je dat toch?
```suggestion
  if (aHasPrecedence == bHasPrecedence) {
    return sortAlphaOnTitle(a, b);
  }
```